zanata-war/src/main/java/org/zanata/dao/ProjectDAO.java

```diff
@@ -433,4 +435,22 @@ public int getMaintainedProjectCount(HPerson maintainer, String filter) {
                         .setParameter("filter", "%" + sqlFilter + "%");
         return ((Long) q.uniqueResult()).intValue();
     }
+
+    public List<HProject> getProjects(EntityStatus... includeStatus) {
```

Comment:
This does almost the same thing as `getOffsetListOrderByName(int offset, int count, boolean filterActive, boolean filterReadOnly, boolean filterObsolete)`, just with offset 0 and no count. Can the two be combined, or just use the existing one with `0` and `Integer.MAX_VALUE`, rather than create another method that we have to maintain?